app/src/main/res/values/integers.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <integer name="grid_layout_column_count">4</integer>
-    <integer name="grid_layout_row_count">4</integer>
+    <integer name="layout_weightValue">1</integer>
```

Comment:
not use cammelcase to define a name in the resource, use snake style

Reply:
Done